director/templates/index.html

```diff
@@ -178,33 +259,31 @@
-  {% else %}
-  <script src="%% url_for('user.static', filename='vue.min.js') %%"></script>
-  <script src="%% url_for('user.static', filename='vue-router.min.js') %%"></script>
-  <script src="%% url_for('user.static', filename='vuex.min.js') %%"></script>
-  <script src="%% url_for('user.static', filename='vuetify.min.js') %%"></script>
-  <script src="%% url_for('user.static', filename='moment.min.js') %%"></script>
-  <script src="%% url_for('user.static', filename='axios.min.js') %%"></script>
-  <script src="%% url_for('user.static', filename='vis-network.min.js') %%"></script>
-  {% endif %}
-  <script src="%% url_for('static', filename='script.js') %%"></script>
-</body>
-</html>
+    </div>
+    <script type="text/javascript">
+      const DARK_THEME=%% config.ENABLE_DARK_THEME | int %%;
+      const API_URL="%% config.API_URL %%";
+      const FLOWER_URL="%% config.FLOWER_URL %%";
+    </script>
+    {% if config.ENABLE_CDN %}
+    <script src="https://cdn.jsdelivr.net/npm/vue@2.6.11/dist/vue.min.js" integrity="sha256-ngFW3UnAN0Tnm76mDuu7uUtYEcG3G5H1+zioJw3t+68=" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vue-router@3.3.4/dist/vue-router.min.js" integrity="sha256-87g98o5+aw7/ExJ9tBjvH8zz46FJ7hiylPtNFcPvsSw=" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vuex@3.1.2/dist/vuex.min.js" integrity="sha256-LfE9mPMjeOg3dTn1sESY2Xvdbq7gAhONtkxacnr7FSA=" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vuetify@2.2.1/dist/vuetify.min.js" integrity="sha256-Wksm4daF3JKKD97f/NHSJkxqQCk9XwujIEBTX0VLzTw=" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/moment@2.24.0/min/moment.min.js" integrity="sha256-4iQZ6BVL4qNKlQ27TExEhBN1HFPvAvAMbFavKKosSWQ=" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/axios@0.19.0/dist/axios.min.js" integrity="sha256-S1J4GVHHDMiirir9qsXWc8ZWw74PHHafpsHp5PXtjTs=" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/d3@5.16.0/dist/d3.min.js" integrity="sha256-Xb6SSzhH3wEPC4Vy3W70Lqh9Y3Du/3KxPqI2JHQSpTw=" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/dagre-d3@0.6.4/dist/dagre-d3.min.js" integrity="sha256-dPm4TA8Y9PY5q5mmtWMkRGOCMHJDKy34ZrxdbBGA9cs=" crossorigin="anonymous"></script>
+    {% else %}
+    <script src="%% url_for('user.static', filename='vue.min.js') %%"></script>
+    <script src="%% url_for('user.static', filename='vue-router.min.js') %%"></script>
+    <script src="%% url_for('user.static', filename='vuex.min.js') %%"></script>
+    <script src="%% url_for('user.static', filename='vuetify.min.js') %%"></script>
+    <script src="%% url_for('user.static', filename='moment.min.js') %%"></script>
+    <script src="%% url_for('user.static', filename='axios.min.js') %%"></script>
```

Comment:
As you add `d3.min.js` and `dagre-d3.min.js`, you have to add these libs into the `else` block too

Reply:
Fixed, thanks